recipes/ms-gsl/all/conanfile.py

```diff
@@ -43,15 +43,15 @@ def _compilers_minimum_version(self):
         }
 
     def config_options(self):
-        if tools.Version(self.version) >= "3.0.0":
+        if Version(self.version) >= "3.0.0":
             del self.options.on_contract_violation
 
     def configure(self):
```

Comment:
Please rename `validate`, because compiler version check should be placed in `validate` not `configure`.

Reply:
I totally missed that. Of course this should use the `validate` method and not `configure`. I've submitted PR #12453 to fix this. Thanks!